src/SqlSquared/Parser.purs

```diff
@@ -676,63 +543,33 @@ tableRelation = PC.try do
        (pure ∘ E.Left ∘ (Pt.rootDir Pt.</> _))
        ∘ Pt.sandbox Pt.rootDir)
       i
-
   a ← PC.optionMaybe do
     _ ← keyword "as"
     ident
   pure $ Sig.TableRelation { alias: a, path }
 
 
-variRelation
-  ∷ ∀ t m
-  . Corecursive t (Sig.SqlF EJ.EJsonF)
-  ⇒ Monad m
-  ⇒ P.ParserT (Array Token) m (Sig.Relation t)
-variRelation = PC.try do
+variRelation ∷ ∀ m t. SqlParser m t (Sig.Relation t)
+variRelation = do
   operator ":"
   vari ← ident
```

Comment:
`create function foo(:in, :range)` is valid expression if we look at parser. Isn't `select * from :in` valid expression too? Maybe tokenizer should extract varis, what do you think?